src/diffusers/models/attention_processor.py

```diff
@@ -1344,6 +1350,13 @@ def __call__(
             split_size = kv.shape[-1] // 2
             key, value = torch.split(kv, split_size, dim=-1)
 
+        # else:
+        #     query = attn.to_q(hidden_states, *args)
+        #     if attn.norm_cross:
+        #         encoder_hidden_states = attn.norm_encoder_hidden_states(encoder_hidden_states)
+        #     key = attn.to_k(encoder_hidden_states, *args)
+        #     value = attn.to_v(encoder_hidden_states, *args)
+
```

Comment:
```suggestion
        # else:
        #     query = attn.to_q(hidden_states, *args)
        #     if attn.norm_cross:
        #         encoder_hidden_states = attn.norm_encoder_hidden_states(encoder_hidden_states)
        #     key = attn.to_k(encoder_hidden_states, *args)
        #     value = attn.to_v(encoder_hidden_states, *args)

```

This PR is still WIP?

Reply:
I think you missed https://github.com/huggingface/diffusers/pull/6181#issuecomment-1871078562